src/libsystemd/sd-bus/bus-objects.c

```diff
@@ -1610,6 +1610,39 @@ static int vtable_member_compare_func(const struct vtable_member *x, const struc
 
 DEFINE_PRIVATE_HASH_OPS(vtable_member_hash_ops, struct vtable_member, vtable_member_hash_func, vtable_member_compare_func);
 
+static bool names_are_valid(const char *signature, const char **names, int *flags) {
+        enum {
+                first           = 1<<0,
+                have_names      = 1<<1,
+                alone           = 1<<2,
+        };
+
+        if((*flags&first || *flags&alone ) && **names)
```

Comment:
whitespace is weird. Please follow our usual coding style, i.e. a space after the `if`, and before and after each operator, but not before `)`